### DISCUSSION TITLE
Web 版本play报错

### DISCUSSION THREAD

**Discussion by Djuvl** [Tencent/libpag#2608] [Q&A] [answered]
报错内容：
Uncaught (in promise) Error: Seek operation timed out.
    at video-reader.ts:288

概率出现 报错后再进行其他pagview.init都无法成功play 也没有再报错
版本为4.3.51

**Comment by Djuvl** [comment]
尝试重新PAGInit 可以再次播放其他pag 但是没找到PAGInit对应的destroy之类的方法 不知道会不会有问题

**Comment by jinwuwu001** [comment]
请基于release最新版验证下该问题是否还存在，若存在请提供一份demo

**Comment by jinwuwu001** [comment, marked as answer]
该问题是视频解码超时抛异常导致播放超时，该问题已修复，可更新至 4.5.47，新版本中解码超时会日志提示，但不会抛异常
